docs/src/content/overview-installation.md

```diff
@@ -114,7 +135,12 @@ pip3 install mitmproxy
 
 ## Installation from Source
 
-If you would like to install mitmproxy directly from source code or the
-GitHub master branch, please see the our
-[README](https://github.com/mitmproxy/mitmproxy/blob/master/README.rst)
-on GitHub.
+Download the [release
+package](https://github.com/mitmproxy/mitmproxy/releases).
+
+{{< highlight bash  >}}
+tar xvzf mitmproxy-[VERSION].tar.gz
+cd mitmproxy-[VERSION]
+pip install .
+{{< / highlight >}}
+
```

Comment:
I'm 👎 on this. I think we do want to point to the README here, so that everyone looking at the source code (no matter by which entrypoint) uses the description in README.rst.